### PR TITLE
Add gijgo@1.3.1-pre-release w/ npm auto-update

### PR DESCRIPTION
**Pull request by atatanasov**
Library name: Gijgo
Git repository url: https://github.com/atatanasov/gijgo/
npm package name or url (if there is one): https://www.npmjs.com/package/gijgo
License (List them all if it's multiple): MIT License
Official homepage: http://gijgo.com

Pull request for issue: #
Related issue(s): https://github.com/cdnjs/cdnjs/pull/11077

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * License and its reference:
 * GitHub / Bitbucket popularity (required):
   - Count of watchers:
   - Count of stars:
   - Count of forks:
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [ ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [ ] The parent of the commit(s) in the PR is not old than 3 days.
 * [ ] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by pvnr0082t** [comment]
Hi @atatanasov ,

This library doesn't have enough notable popularity(like more than 100 stars on GitHub or 500 downloads per month on npm). please take a look on our [README.md](https://github.com/cdnjs/cdnjs/blob/master/README.md) for detail.

cc @PeterDaveHello

**Comment by PeterDaveHello** [comment]
@pvnr0082t we can accept this one, see #10799

**Comment by pvnr0082t** [comment]
@PeterDaveHello Yeah, I just saw that.  Thanks.  :stuck_out_tongue:

File: ajax/libs/gijgo/package.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
+  "name": "gijgo",
+  "version": "1.3.1",
+  "description": "Gijgo is a set of free open source javascript controls distributed under MIT License. All widgets are high performance, built on top of the jQuery JavaScript Library and with built-in support for Bootstrap and jQuery UI. They are designed to saves you time and scales with your development process.",
+  "main": "js/gijgo.js",
```
Comment (pvnr0082t) [inline ajax/libs/gijgo/package.json]: @atatanasov We don't need `main`, `bugs`, `scripts`, `dependencies` and `devDependencies` field. Please remove them. Thanks.

**Comment by PeterDaveHello** [comment]
Move to #11110

**Pull request closed without merging**